_source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md

```diff
@@ -13,21 +13,28 @@ contributors:
   - yberlinger
 ---
 
-Grafana Explore is where you can explore the data available in your Metrics account.
+Grafana Explore is where you can explore the data available in your Prometheus metrics account.
 It's a bit like Kibana Discover, in that it is optimized for quickly searching the data in preparation for creating dashboards.
 
 Whether you just started sending metrics for the first time, or you want to check that your metrics arrived as expected, Grafana’s Explore mode is the best way to do it.
 It's also great if you're a long-time user and want to examine the structure of your metrics documents in order to create a new monitoring dashboard.
```

Comment:
```suggestion
It's also great if you're a long-time user and want to examine the structure of your metrics in order to create a new monitoring dashboard.
```

Reply:
Now we know P8S metrics are not arranged in docs anymore, as opposed to Elastic.